src/rocksdb.rs

```diff
@@ -1000,6 +1024,20 @@ impl Writable for DB {
     fn single_delete_cf(&self, cf: &CFHandle, key: &[u8]) -> Result<(), String> {
         self.single_delete_cf_opt(cf, key, &WriteOptions::new())
     }
+
+    fn delete_range(&self, begin_key: &[u8], end_key: &[u8]) -> Result<(), String> {
+        let batch = WriteBatch::new();
```

Comment:
@BusyJay See the previous comment
https://github.com/pingcap/rust-rocksdb/pull/20#commitcomment-21151655
I don't see any efficiency issue here since the code of RocksDB implements DB's interfaces using `WriteBatch`.